sinonjs/referee
# Fix browser compat

This PR fixes an issue with browser compatibility

#### Background

Because of a missing configuration of `rollup`, `referee` has NOT worked outside of node for a very long time.

There were no shims installed for these two dependencies in the `rollup` config:

```
lib/referee.js:3:27
var EventEmitter = require("events");

lib/format.js:3:22
var inspect = require("util").inspect;
```

#### Solution

1. Add engine neutral shims for EventEmitter and `util` to improve compatibility
1. Replace `rollup` with `esbuild` (the last two commits can be dropped if this undesirable)

#### How to verify - mandatory

1. Check out this branch
2. `npm install`
1. `npm run build`
3. `npm link`
1. In a project using `@sinonjs/referee`, run `npm link @sinonjs/referee`
1. Change the import to use the bundled file in `dist/`
1. Run tests in the consuming project
1. Observe tests pass as normal

#### Checklist for author
